airflow/www/api/experimental/endpoints.py

```diff
@@ -74,8 +77,16 @@ def trigger_dag(dag_id):
 
             return response
 
+    # Default replace_microseconds to True follow convention from trigger.trigger_dag
+    replace_microseconds = True
```

Comment:
I think we could simplify it like below
```suggestion
    replace_microseconds = True
    if 'replace_microseconds' in data and data['replace_microseconds'].lower() not in YES_VALUES:
        replace_microseconds = False
```